build/projects/AI Test Toolkit/.AL-Go/settings.json

```
@@ -0,0 +1,17 @@
+{
+    "projectName": "AI Test Toolkit",
+    "appFolders":  [
+        "../../../src/Tools/AI Test Toolkit/App"
```

```suggestion
        "../../../src/Tools/AI Test Toolkit"
```

The current folder doesn't exist and the app is not getting build: https://github.com/microsoft/BCApps/actions/runs/9812717511/job/27097354877#step:8:88